test/integration/targets/vmware_guest/tasks/reconfig_vm_to_latest_version.yml

```diff
@@ -3,13 +3,59 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 # Skipping out idepotency test untill issue fixed in reconfigure_vm() become_method
 
+- name: Create VM with hardware version 13
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    name: test_vm_version12
+    guest_id: centos7_64Guest
+    datacenter: "{{ dc1 }}"
+    folder: "{{ f0 }}"
+    datastore: '{{ ds2 }}'
+    hardware:
+        num_cpus: 4
+        memory_mb: 1028
+        version: 12
```

Comment:
This is 13?

Reply:
Mismatch in testcase name and version used. Fixed it.
Thanks for catching this